packages/medusa/src/api/routes/admin/orders/__tests__/create-fulfillment.js

```diff
@@ -42,7 +43,7 @@ describe("POST /admin/orders/:id/fulfillment", () => {
             quantity: 1,
           },
         ],
-        undefined
+        {"metadata": undefined, "noNotification": undefined}
```

Comment:
Change required since we now capture both metadata and noNotification within a config object

Reply:
please snake_case this badboy